gql/models/UserJobApplication/resolvers/index.js

```diff
@@ -6,7 +6,7 @@ import { getUsersByIds } from '../../User/cdb';
 export default {
   UserJobApplication: {
     job: (jobApplication) => getJobsByIds(jobApplication.jobId),
-    user: (jobApplication) => getUsersByIds(jobApplication.id),
+    user: (jobApplication) => getUsersByIds(jobApplication.userId),
```

Comment:
Hey @phoenixdo , I don't see any new query being added and the line you changed is the same as master. I am missing something ?